priv/perf/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: list
+list:
+	@$(MAKE) -pRrq -f $(lastword $(MAKEFILE_LIST)) : 2>/dev/null | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}' | sort | egrep -v -e '^[^[:alnum:]]' -e '^$@$$'
+
+services_up:
```

Comment:
This looks similar to [`make start-services` in the repo root](https://github.com/omisego/elixir-omg/blob/master/Makefile#L371). Maybe rename to `start-services`, and `services_down` to `stop-services`?